docs/proposals/Avatar/README.md

```diff
@@ -0,0 +1,78 @@
+# Avatar
+
+An `<Avatar />` component will be used to display an identifier (image or
+initials) for a user.
+
+Rebecca Li has done some
```

Comment:
Some parts of that proposal are being implemented here, and some aren't. That's fine. But, any thoughts on how to keep that design document relevant as we continue to develop the component? Maybe link it in the actual component Readme?

Reply:
That would be a good start. Could we add a ticket to the Frends backlog capturing the other components (avatarGroup, teamAssignmentChecklist)?

I'm also fairly certain the Manage Team work will require the List variation of this component from the proposal
![image](https://user-images.githubusercontent.com/39704901/80037166-54cc3100-84b0-11ea-80e8-a1d1e7babcd8.png)

Reply:
With Atlantis being open source, @chris-at-jobber can we link to the design documents from the Readme? Im also wondering if I even should have put this in the proposal now.

@chris-at-jobber in regards to the `single line` and `2 line` displays, I don't think that these are the responsibility of the avatar. I would say that the avatar is only responsible for the display of the initials and/or image.

Reply:
It's my understanding that linking a figma file should grant only access to that file. Since it's relevant to this I _think_ it's fine. Perhaps something we should think about further for the future though.

Reply:
Let me look into some Figma permissions things - it might require a more involved account setup for engineers than current, but it should be feasible to say "Anyone at Jobber can view"